### ISSUE TITLE
Add documentation on using specific lftp commands to Savio <-> Box doc

### ISSUE THREAD

**Issue by aronr** [ucberkeley/brc-draft-documentation#35]
In https://github.com/ucberkeley/brc-draft-documentation/wiki/Transferring-Data-Between-Savio-and-Your-UC-Berkeley-Box-Account, add at least summary information on performing file transfers with the `lftp` tool.

We might at least cover `ls`, `cd`, `get`, `mget`, `put`, and `mput`.

Per Wynn Meyer, `mirror` is also powerful and useful:
> The command to transfer a directory recursively (including everything inside it)
> is just 'mirror <directory>' for downloading from box, and 'mirror -R <directory>'
> for uploading to box.

Note that the 'mirror' command involves some element of risk: of overwriting existing files where that isn't desired. There are many options to this command, and perhaps we might recommend some combination and/or best practices for minimizing that risk

**Comment by aronr** [comment]
Some of this documentation has been added (with thanks to @smasover). It could benefit by some user testing.

Also, we might yet consider the possible addition of a (careful) mention of 'mirror', as per above.

**Comment by paciorek** [comment]
I agree mirror would be quite useful but see the point about caution. Perhaps add mention of mirror with a caveat emptor about overwriting.
